src/zope/site/site.rst

```diff
@@ -352,3 +352,9 @@ sitemanager's bases should be set to global site manager.
   >>> nosm['root'] = myfolder11
   >>> myfolder11.getSiteManager().__bases__ == (gsm, )
   True
+
+Deleting a site unregisters its site manger from its parent site manager:
+
+  >>> del myfolder2['myfolder21']
+  >>> myfolder2.getSiteManager().subs
+  ()
```

Comment:
It's probably worth testing the `__bases__` explicitly?

```suggestion
  ()
  
The removed site manager now has no bases:

  >>> myfolder21.getSiteManager().__bases__
  ()
```

Reply:
You are right, we should test it explicitly.